src/lib.rs

```diff
@@ -2,7 +2,7 @@
 #![deny(clippy::undocumented_unsafe_blocks)]
 #![deny(unsafe_op_in_unsafe_fn)]
 #![doc = include_str!("../README.md")]
-pub extern crate anymap;
+extern crate anymap3;
```

Comment:
I'm not sure if the `pub` is really necessary here. Since this change is already breaking the public API (`generic_singleton::anymap` won't exist), I thought it would be OK to stop exporting it at all (so that similar changes later won't break the public API).

Another solution that wouldn't break the API would be

```
pub extern crate anymap3;
pub use anymap3 as anymap;
```